### PR TITLE
[GSoC Proposal] ScienceBox Monitoring, SWAN, CERN

### PR DESCRIPTION
**Pull request by ebocchi**

### CODE REVIEW

File: _gsocprojects/2020/project_SWAN.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+project: SWAN
+layout: default
+logo: swan_logo.svg
```
Comment (agheata) [inline _gsocprojects/2020/project_SWAN.md]: We have a different logo naming convention now: SWAN-logo.svg. Please copy images/swan_logo.svg  to  images/SWAN-logo.svg and use the new name

Comment (agheata) [reply _gsocprojects/2020/project_SWAN.md]: Pls do not delete the old logo file, it is in use. If you can provide .png instead of .svg even better

Comment (agheata) [reply _gsocprojects/2020/project_SWAN.md]: @ebocchi you may need to rebase on master and fix the conflicts on mentors.md before re-pushing

File: _gsocprojects/2020/proposal_ScienceBoxMonitor.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (aflp91) [inline _gsocprojects/2020/proposal_ScienceBoxMonitor.md]: This proposal file should be in `_gsocproposals/2020` directory…

File: _gsocprojects/2020/project_SWAN.md (hunk 1 above)
Comment (ebocchi) [reply _gsocprojects/2020/project_SWAN.md]: It should be fixed now. Thanks!

File: _gsocprojects/2020/proposal_ScienceBoxMonitor.md (hunk 2 above)
Comment (ebocchi) [reply _gsocprojects/2020/proposal_ScienceBoxMonitor.md]: It is fixed now, sorry

**Comment by ebocchi** [comment]
It should be good now. Hope it is not too late for merging.
Thanks a lot!

**Pull request merged**